docs/admin/install/knative-with-operators.md

```diff
@@ -21,6 +21,12 @@ kubectl apply -f {{artifact(org="knative",repo="operator",file="operator.yaml" )
 
 Verify your installation:
 
+Be sure to set the current namespace to the `default` as the operator is installed to `default` namespace:
+```
+kubectl config set-context --current --namespace=default 
+```
```

Comment:
```suggestion
1. Because the operator is installed to the `default` namespace, ensure you set the current namespace to `default` by running the command:

    ```
    kubectl config set-context --current --namespace=default 
    ```
```